tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.md

```diff
@@ -0,0 +1,83 @@
+Test 5-22 - NFS Volume
```

Comment:
@mhagen-vmware I can confirm this, right now we do not pass additional mount options or share options through. It is possible(and very likely) that we will want to implement mount options per `VolumeJoin` which would be at `docker create` time. For now we do not, and if you create a `ReadOnly` share point we will fail at install since the portlayer requires the ability to create some namespacing as well as write metadata.